### ISSUE TITLE
3.0/develop missing dependencies

### ISSUE THREAD

**Issue by foag** [helsingborg-stad/Municipio#195]
**Branch:** 3.0/develop

**Environment:** macOs, Wordpress 5.7.2, Nginx

**Issue:** 
`Fatal error: Uncaught Error: Class 'ComponentLibrary\Init'`
`/themes/municipio/library/Template.php on line 16`

Steps to reproduce: 
Clone fresh repo into a local WP themes directory. Install npm/composer dependencies. Activate theme. Boom goes WP.

**Comment by sebastianthulin** [comment]
Install https://github.com/helsingborg-stad/component-library. This is a required component to run 3.0 branch.

**Comment by foag** [comment]
> Install https://github.com/helsingborg-stad/component-library. This is a required component to run 3.0 branch.

I did install this after some googling, although PHP/WP doesn't seem to recognize the ComponentLibrary\Init class.

**Comment by sebastianthulin** [comment]
There might be an loading issue. It might sound wierd, but try to install Modularity 3.0 branch, loading should be fixed there.

**Comment by foag** [comment]
Good idea, but unfortunately no luck. When switching to _3.0/develop-modularity-compability_, rimraffing vendor and doing a fresh `composer install` I get a version of the same error, only this time pointing to `BladeComponentLibrary\Init` instead of the original `ComponentLibrary`.

**Comment by sebastianthulin** [comment]
There might be multiple culprits here. Try to: 

- Composer install in all of the plugins (or php build.php). 
- Move the Component Library plugin to the mu-plugins folder to enshure that i loads first. This requires a root-file in the mu-plugins directory including main php file of component library ( component-library.php ).

**Comment by foag** [comment]
Ah, that did the trick. When installing the component-library from the theme root a wild wp-content folder appeared in the theme root. Moving this to mu-plugins and setting up a loader fixed it. 

This should be documented as a preface to installing Municipio though - or have I missed some crucial documentation?
